.github/workflows/functional.yml

```diff
@@ -70,11 +73,60 @@ jobs:
       - name: Bootstrap/Setup RSTUF using DAS and run Functional Tests
         timeout-minutes: 30
         run: |
-          docker compose run --env UMBRELLA_PATH=. --rm rstuf-ft-runner bash tests/functional/scripts/run-ft-das.sh ${{ inputs.cli_version }}
+          docker compose run --env UMBRELLA_PATH=. --rm rstuf-ft-runner bash tests/functional/scripts/run-ft-das.sh ${{ inputs.cli_version }} ${{ matrix.pytest-group }}
```

Comment:
What will happen when you pass the `${{ matrix.pytest-group }}` to docker compose?

Reply:
It's actually being passed to `tests/functional/scripts/run-ft-das.sh` (as we did also with `inputs.cli_version`) and from that bash script we call the `Makefile` along with those parameters ([here](https://github.com/repository-service-tuf/repository-service-tuf/blob/7f004af8bb3026f505ef464e191f1524add2c221/tests/functional/scripts/run-ft-das.sh#L93)).
In the end, what's being called is: `pytest --exitfirst --splits 3 --group $(PYTEST_GROUP) ...` and the `${{ matrix.pytest-group }}` you asked about is the parameter that defines the testing group that is being run at that moment. `pytest-split` splits the tests in groups and on every pytest run you define which group you run.